content/geoip/importing-databases/postgresql.mdx

```diff
@@ -86,19 +86,18 @@ We can now import the contents of `GeoIP2-City-Blocks-IPv4.csv` and
 `GeoIP2-City-Blocks-IPv6.csv` into the table we just created:
 
 ```sql
-\copy geoip2_network(
-  network, geoname_id, registered_country_geoname_id, represented_country_geoname_id,
-  is_anonymous_proxy, is_satellite_provider, postal_code, latitude, longitude, accuracy_radius, is_anycast
-) from 'GeoIP2-City-Blocks-IPv4.csv' with (format csv, header);
+\copy geoip2_network from 'GeoIP2-City-Blocks-IPv4.csv' with (format csv, header);
 ```
 
 ```sql
-\copy geoip2_network(
-  network, geoname_id, registered_country_geoname_id, represented_country_geoname_id,
-  is_anonymous_proxy, is_satellite_provider, postal_code, latitude, longitude, accuracy_radius, is_anycast
-) from 'GeoIP2-City-Blocks-IPv6.csv' with (format csv, header);
+\copy geoip2_network from 'GeoIP2-City-Blocks-IPv6.csv' with (format csv, header);
 ```
 
+If there are errors such as `missing data for column` or
+`extra data after last expected column`, alter the table to meet the CSV file
+structure. For example, GeoLite2-City CSV file doesn't contain `is_anycast`, so
```

Comment:
GeoLite2-City is going to have that column actually. We could drop this last sentence I think.